circle.yml

```diff
@@ -0,0 +1,17 @@
+dependencies:
+  cache_directories:
+    - /usr/local/Homebrew
```

Comment:
I think it's only caching the work done in `dependences > override`, e.g. caching `shellcheck`. Maybe worth removing this for clarity.